client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -84,10 +86,21 @@ export const ApplicationDetailDrawer: React.FC<
           <TextContent className={`${spacing.mtMd} ${spacing.mbMd}`}>
             <Text component="small">{application?.description}</Text>
             {application ? (
-              <Link to={getIssuesSingleAppSelectedLocation(application.id)}>
+              <Link
+                to={getIssuesSingleAppSelectedLocation(application.id)}
+                style={{ display: "block" }}
+              >
                 Issues
               </Link>
             ) : null}
+            {application ? (
+              <Link
+                to={getDependenciesUrlFilteredByAppName(application?.name)}
+                style={{ display: "block" }}
+              >
+                Dependencies
+              </Link>
+            ) : null}
```

Comment:
I'd suggest to use a [plain List](https://www.patternfly.org/components/list#plain) here instead of a `style={{ display: "block" }}` to align vertically.

You could put the `List` with both links within the same `{application ? ...List... : null}` block.